obspy/core/tests/test_stream.py

```diff
@@ -2494,15 +2494,23 @@ def test_read_no_check_compression(self):
         st = read()
         st.write(file_name, 'mseed')  # maybe read from test data instead?
```

Comment:
Yep, would be good to avoid the write operation.. just make use of a short uncompressed ascii format test file below when reading:

```python
read('/path/to/slist.ascii', format='SLIST')
```